README.md

```diff
@@ -99,7 +99,6 @@ To use [rkt](https://github.com/coreos/rkt) as the container runtime run:
 $ minikube start \
     --network-plugin=cni \
     --container-runtime=rkt \
-    --iso-url=http://storage.googleapis.com/minikube/iso/buildroot/minikube-v0.0.6.iso
 ```
 
 This will use an alternative minikube ISO image containing both rkt, and Docker, and enable CNI networking.
```

Comment:
It won’t use an "alternative minikube ISO image" anymore as the minikube-iso will be the default image.